### DISCUSSION TITLE
turn off full-packet capture ?

### DISCUSSION THREAD

**Discussion by 73F0SG1R** [Security-Onion-Solutions/securityonion#5929] [Unsupported Versions] [answered]
the online docs aren't super clear, is the correct way to turn off full packet capture to:

enter:

"steno:enabled:false"

in the "global" part of in this file?:

/opt/so/saltstack/local/pillar/global.sls

thank you

**Comment by dougburks** [comment, marked as answer]
I've updated the documentation to hopefully make it more clear:
https://docs.securityonion.net/en/2.3/stenographer.html#disabling

Hope that helps!

**Reply by 73F0SG1R** [reply]
thank you sir
